python_examples/intervention_example/README.md

```diff
@@ -0,0 +1,45 @@
+# Intervention Examples
+
+Contained within the `intervention_example/` directory, these scripts runs simulations in which one intervention is active. Output graphs are contained within the `intervention_example/intervention_outputs/` directory. All simulations are run using the same input file containing a population of 10000 individuals, spread across 200 cells, each with 2 microcells, and 5 households per microcell. Simulations are run until day 100 with one initial infection. Of all interventions only Social Distancing, Vaccination, and Travel Isolation use age in the model. All interventions consider household, place and spatial infections. Unless mentioned otherwise, intervention parameters are set to the CovidSim parameters (file `p_PC7_CI_HQ_SD.txt`).
+
+## Case Isolation
+
+The resulting infection curve shows the progression of the disease when symptomatic infected cases isolate with different isolation probabilities. Infection curves when none of the symptomatic individuals isolate ('isolation_probability' equals 0.0), half of the symptomatic individuals isolate ('isolation_probability' equals 0.5), and all symptomatic individuals isolate ('isolation_probability' equals 1.0) are compared. The case isolation will last for 10 days. Increasing the isolation probability results in a delayed wave with reduced peak of infections.
+
+![case isolation.pt](./intervention_example/intervention_outputs/case_isolation_Icurve_plot.png)
```

Comment:
```suggestion
![case isolation.pt](./intervention_outputs/case_isolation_Icurve_plot.png)
```

Reply:
Good spot, sorry I missed that. I have updated them and they render correctly online now. Will merge this.